unicodetools/org/unicode/tools/emoji/EmojiData.java

```diff
@@ -330,37 +335,43 @@ private EmojiData(VersionInfo version) {
 		int debug = 0;
 	    }
 	}
-	modifierBasesRgi = new UnicodeSet(modifierBases).removeAll(MULTIPERSON);
-	if (version.compareTo(Emoji.VERSION12) >= 0) {
-	    modifierBasesRgi.addAll(HOLDING_HANDS_COMPOSITES);
-	}
-	if (!modifierBases.contains(0x1F90C)) {
-	    if (version.compareTo(Emoji.VERSION13) >= 0) {
-		int debug = 0;
+	if (oldFormat) {
+	    // Modified this for new format to be driven just from the data files.
+	    modifierBasesRgi = new UnicodeSet(modifierBases).removeAll(MULTIPERSON);
+	    if (version.compareTo(Emoji.VERSION12) >= 0) {
+		modifierBasesRgi.addAll(HOLDING_HANDS_COMPOSITES);
+		if (version.compareTo(Emoji.VERSION13_1) >= 0) {
+		    modifierBasesRgi.addAll(OTHER_GROUP);
+		}
 	    }
-	}
-	modifierBasesRgi.freeze();
-	modifierSequences = new UnicodeSet();
-	for (String base : modifierBasesRgi) {
-	    if (UnicodeSet.getSingleCodePoint(base) == 0x1F46B) {
-		int debug = 0;
+	    if (!modifierBases.contains(0x1F90C)) {
+		if (version.compareTo(Emoji.VERSION13) >= 0) {
+		    int debug = 0;
+		}
 	    }
-	    for (String mod : MODIFIERS) {
-		final String seq = base + mod;
-		modifierSequences.add(seq);
-		// names.put(seq, UCharacter.toTitleFirst(ULocale.ENGLISH, getName(base, true))
-		// + ", " + shortName(mod.codePointAt(0)).toLowerCase(Locale.ENGLISH));
+	    modifierBasesRgi.freeze();
```

Comment:
I revised the code to take the modifier RGI sequences from the data, instead of computing it (which was hacky).